documents/User-Guide.md

```diff
@@ -0,0 +1,131 @@
+
+The mojaloop Self Testing Toolkit was designed for participants that would like to partcipate in the mojaloop Hub scheme. Intentially build to have a standard way to test integration with a Mojaloop Hub, this tool set can potentially be used by both the DFSP and the Hub to verify the inergration between the 2 entities.
+
+For additional back ground information on the Self Testing Toolkit, please see [mojaloop Self Testing Toolkit](/documents/Mojaloop-Self-Testing-Tool.md). It would also be advisable to look at the [Architecture Diagram](/documents/Mojaloop-Self-Testing-Tool.md#7-architecture) to understand the components and related flow.
+
+### 2. Getting Started
+
+To get started, please follow instruction in the [README](/README.md) document. This document covers the use-cases with the mojaloop simulator and mojaloop simulator ui.
+
+### 3. The Self-Testing Tool
+
+#### 3.1 At first glance
+
+When you open the mojaloop Self Testing Toolkit in your browser, you will be welcomed by the Dashboard display. Currently this is still under developement, but the "static data" will provide you with a fair representations of the intention of the Dashboard display.
+
+![Opening view](/assets/images/Opening_View.png)
+
+You would also notice the navigation items on the left. Currently these consist of;
+
+- Dashboard
+- Monitoring
+- Sync Response Rules
+- Validation Rules (Error Callbacks)
+- Callback Rules (Success Callbacks)
+- Outbound Request
+- Settings
+
+We will work through each one of the items and provide you with a fair understanding of the functionality.
+
+![Menu Items](/assets/images/Menu_Items.png)
+
+#### 3.2 Dashboard
+
+This is the default opening screen. As mentioned, this is still being developed. The intent is to provide information to the user of the mojaloop Self Testing Toolkit on Resources Hits(Incoming and Outgoing), Request and Responses statistics and Rules Engine Hits statistics.
+
+More information would be disclosed once the developement is at a more mature stage.
+
+#### 3.3 Monitoring
+
+Allows you to monitor the incomming and outgoing request.
+
+![Monitoring Initial State](/assets/images/Monitoring_Initial_State.png) 
+
+If you have followed the installation in structions, you should have the Simulator UI open in a browser tab. Go to the **Outbound Send** tab on the left.
+
+Press **Send Transfer** button to send a request from the Simulator UI to the Self Testing Toolkit.
+
+![Send Transfer](/assets/images/Send_Transfer.png)
+
+
+If the request was send successfully from the Simulator UI.
+
+![Simulator response](/assets/images/Simulator_Response.png) 
+
+Go back to the Monitoring tab on the Self Testing Toolset. You will notice the three processes assosiated with this request in acending order.
```

Comment:
three processes assosiated with this request in acending order -> three requests associated with a transfer in an order such that most recent request first